src/main/bash/sdkman-install.sh

```diff
@@ -67,7 +67,7 @@ function __sdkman_install_candidate_version() {
 	__sdkman_download "$candidate" "$version" || return 1
 	__sdkman_echo_green "Installing: ${candidate} ${version}"
 
-	mkdir -p "${SDKMAN_CANDIDATES_DIR}/${candidate}"
+	mkdir -p "${SDKMAN_CANDIDATES_DIR}/${candidate}/${version}"
```

Comment:
I don't think this is a good idea. This statement is only here to ensure that the candidates folder exists. Creation of the version directory happens when we move the archive contents here from the temp folder.